nameserver/server.go

```diff
@@ -33,16 +30,47 @@ type DNSServerConfig struct {
 	Port int
 	// (Optional) local domain (ie, "weave.local.")
 	LocalDomain string
+	// (Optional) cache size
+	CacheLen int
+	// (Optional) number of resolution workers for local queries
+	NumLocalWorkers int
```

Comment:
Oh, `NumLocalWorkers` does not really belong to this PR but to the next one... ;)